draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -340,37 +354,59 @@
           configured with an algorithm to combine the recognized metrics
           carried by the Sub-TLVs within a Metadata Path Attribute of the
           received BGP UPDATE message.</t>
+	  <!-- XXX JMH - This text is really an error handling consideration.
+	  Perhaps consider renaming the section to note that? -->
```

Comment:
Do you mean the following text from the Section 7 (Validation and Error Handling) is the repeat of the Section 4.1.2 (Metadata Path Attribute Processing)? which wording do you like better? 

_Suppose a Metadata Path attribute can be parsed correctly but contains a Sub-
TLV whose type is not recognized by a particular BGP speaker; that BGP speaker MUST NOT
consider the attribute malformed. Instead, it MUST interpret the attribute as if that Sub-TLV had
not been present. Logging the error locally or to a management system is optional. If the route
carrying the Metadata path attribute is propagated with the attribute, the unrecognized Sub-TLV
remains in the attribute._

Reply:
What I'm saying here is that the section title says "sub-tlvs data processing" and the only thing discussed is error cases.

Reply:
I see your point. There was a long section on BGP custom decision on combining all the path attributes, local policies and the newly proposed metadata in previous version. It was recommended to move to the appendix. 

How about adding the following sentence that summarize the Custom Decision that was moved to the Appendix? 

_"When an ingress node receives a BGP Update message containing the newly proposed Metadata Path Attribute, it will carefully evaluate the entire path, considering existing path attributes, local policies, and the information carried by the Metadata. To ensure consistent route selection, a deployment-specific algorithm should be configured across all ingress nodes to factor in the Metadata's contribution alongside existing policies. This will help the ingress node make informed decisions about the optimal path to the next-hop, considering both traditional routing factors and the additional insights provided by the Metadata."_

Reply:
That helps.  Marking this comment as resolved.